client/src/app/access/login/login.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+
+import { AuthService } from '@auth0/auth0-angular';
+
+@Component({
+  selector: 'app-login',
+  template: `
+    <button (click)=loginWithRedirect() class="btn-dark">Log in</button>
+  `,
+})
+export class LoginComponent {
+  constructor(private authService: AuthService) {}
+
+  loginWithRedirect() {
```

Comment:
The naming convention I tend to follow with Angular event handlers is using an `on` prefix.

That way, when viewing the list of methods in a component you know which are event handlers very quickly.

So, I would name this `onLoginClick`.